apps/freelance/src/redux/profileQuestions/profileQuestions1Api.ts

```diff
@@ -0,0 +1,41 @@
+import { createApi, fetchBaseQuery } from '@reduxjs/toolkit/query/react';
+
+export const profileQuestions1Api = createApi({
+  reducerPath: 'profileQuestions1Api',
+  baseQuery: fetchBaseQuery({ baseUrl: 'http://localhost:5000' }),
```

Comment:
today you have localhos, tomorrow https://mysuperwebsite you will all the time change URL?
Please read what it env variables are, why we need to use it, and when. Create the list for me: in which case you will use env, in which constants, and when you will write something directly in the code, and send it to me in private message, without this work I will not accept your PR)

Reply:
fixed